tests/swe_lun_occult.phpt

```diff
@@ -70,7 +70,7 @@ array(3) {
   ["attr"]=>
   array(8) {
     [0]=>
-    float(86.757454)
+    float(86.757441)
```

Comment:
Oddly this test passes in my local build but not in Github. I will fix it later.